config: update the self metrics after UTF-8 support was added

ridwanmsharif opened the pull request:
Corresponding Dashboard PR: https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/1146

dashpole commented:
This is probably because we switched to using OTLP to export instead of scraping our own self-obs metrics

ridwanmsharif commented:
> This is probably because we switched to using OTLP to export instead of scraping our own self-obs metrics

Yeah thats right. We still would've needed to update atleast the metric names though since the prometheus scrapes would now include the UTF-8 chars. Updated the title to be more clear
